packages/core/lib/match.js

```diff
@@ -10,8 +10,6 @@ module.exports = match
  *
  * @param {string|RegExp|Function|Array} test   A glob like `*.css` or `{*.js, *.jsx}` or something else to use as `loader.test`.
  * @param {object} [options]                    Optional advanced matching options.
- * @param {string|Function|RegExp|Array|object} [options.include]
```

Comment:
If we remove those hints, then we definitely need to improve the documentation of the overall `options` parameter, since otherwise it's not clear anymore what kind of options can be passed here 😉